prov/usnic/src/usdf_fabric.c

```diff
@@ -309,7 +316,7 @@ usdf_fill_info_dgram(
 
 		fi->handle = hints->handle;
 	} else {
-		fi->mode = USDF_DGRAM_SUPP_MODE;
+		fi->mode = USDF_DGRAM_REQ_MODE;
```

Comment:
Yes, it used to be `USDF_DGRAM_SUPP_MODE` which is defined as: 

```
#define USDF_DGRAM_SUPP_MODE (FI_LOCAL_MR | FI_MSG_PREFIX | FI_MSG_PREFIX2)
```
But now I'm returning `USDF_DGRAM_REQ_MODE`:
```
 #define USDF_DGRAM_REQ_MODE (FI_LOCAL_MR)
```
So I guess I changed default behavior to not return either one unless explicitly asked for.